### DISCUSSION TITLE
function DIVIDE() return_type never DECIMAL ?

### DISCUSSION THREAD

**Discussion by jimdickson007** [duckdb/duckdb#14933] [Q&A] [answered]
`SELECT  function_name, function_type, return_type, parameter_types FROM duckdb_functions() where function_name in ('/','//','divide') ORDER BY function_name, parameters ;`

![image](https://github.com/user-attachments/assets/f9203b9e-b8c3-4d6b-ad40-4711ec2dde94)

Qu: if I want division of DECIMAL parameters to return DECIMAL, do I always need to CAST ?

**Comment by soerenwolfers** [comment, marked as answer]
Yes. It'll remind you that you're not doing exact calculations anymore, which is what people usually use DECIMALs for.

**Reply by soerenwolfers** [reply]
Now documented via https://github.com/duckdb/duckdb-web/pull/4193
